examples/ucnf-kiknos/Makefile

```diff
@@ -0,0 +1,161 @@
+# Parent image for the NSE
+VPP_AGENT = ciscolabs/kiknos:latest
+# Set the org of new built image
+ORG = tiswanso
```

Comment:
nit: I always use `?=` to set org and tag in makefiles so folks can test with their own settings and still use the makefiles.

Reply:
I wanted to use it like this at first, but for some reason when using targets included from other files, the default values from `docker-targets.mk` are taken instead of the ones defined here, and I am unable to execute some rule such as `k8s-ucnf-kiknos-save`.

 Because of the `include`, there happens some magic and I can't understand why it doesn't work with `?=` 🤷 

I tried defining them in many different ways but only `=` worked ok for most usecases.

Reply:
It won't work with `?=` because all the include stuff happens first so the top level makefile setting takes over.  That's why the change in the vl3 NSE Makefile needs to just use a different var or the value directly on the docker build command line.  If/when we clean this up we can get rid of a lot of this Makefile macro "magic" to make it clear.